apps/workflows/jobs/refill-daily.ts

```diff
@@ -44,36 +67,44 @@ client.defineJob({
             remaining: key.refillAmount,
             lastRefillAt: new Date(),
           })
-          .where(eq(schema.keys.id, key.id));
+          .where(eq(schema.keys.id, key.id))
+          .catch((error) => {
+            console.error(`Failed to update remaining ${key.id}`);
+            console.error(`With error ${error}`);
+          });
       });
 
       await io.runTask(`create audit log refilling ${key.id}`, async () => {
-        await tb.ingestAuditLogs({
-          workspaceId: key.workspaceId,
-          event: "key.update",
-          actor: {
-            type: "system",
-            id: "trigger",
-          },
-          description: `Refilled ${key.id} to ${key.refillAmount}`,
-          resources: [
-            {
-              type: "workspace",
-              id: key.workspaceId,
+        await tb
+          .ingestAuditLogs({
+            workspaceId: key.workspaceId,
+            event: "key.update",
+            actor: {
+              type: "system",
+              id: "trigger",
             },
-            {
-              type: "key",
-              id: key.id,
+            description: `Refilled ${key.id} to ${key.refillAmount}`,
+            resources: [
+              {
+                type: "workspace",
+                id: key.workspaceId,
+              },
+              {
+                type: "key",
+                id: key.id,
+              },
+            ],
+            context: {
+              location: "trigger",
             },
-          ],
-          context: {
-            location: "trigger",
-          },
-        });
+          })
+          .catch((_error) => {
+            console.error(`failed to update Audit Logs for ${key.id}`);
```

Comment:
is console.error logging to triggers dashboard?

Reply:
I think it was just to have some error. I assume this will still work with just a throw error??